resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -47,120 +47,30 @@ package object config extends Logging {
-        " this should be the exact string value of the token to use for the authentication. This" +
-        " token value is mounted as a secret on the driver pod.")
-      .stringConf
-      .createOptional
-
-  private[spark] val KUBERNETES_DRIVER_MOUNTED_CA_CERT_FILE =
-    ConfigBuilder(s"$APISERVER_DRIVER_CONF_PREFIX.mounted.caCertFile")
-      .doc("Path on the driver pod's disk containing the CA cert file to use when authenticating" +
-        " against Kubernetes. Typically this is configured by spark-submit from mounting a" +
-        " secret from the submitting machine into the pod, and hence this configuration is marked" +
-        " as internal, but this can also be set manually to use a certificate that is mounted" +
-        " into the driver pod via other means.")
-      .stringConf
-      .createOptional
-
-  private[spark] val KUBERNETES_DRIVER_MOUNTED_CLIENT_KEY_FILE =
-    ConfigBuilder(s"$APISERVER_DRIVER_CONF_PREFIX.mounted.clientKeyFile")
-      .doc("Path on the driver pod's disk containing the client key file to use when" +
-        " authenticating against Kubernetes. Typically this is configured by spark-submit from" +
-        " mounting a secret from the submitting machine into the pod, and hence this" +
-        " configuration is marked as internal, but this can also be set manually to" +
-        " use a key file that is mounted into the driver pod via other means.")
-      .internal()
-      .stringConf
-      .createOptional
-
-  private[spark] val KUBERNETES_DRIVER_MOUNTED_CLIENT_CERT_FILE =
-    ConfigBuilder(s"$APISERVER_DRIVER_CONF_PREFIX.mounted.clientCertFile")
-      .doc("Path on the driver pod's disk containing the client cert file to use when" +
-        " authenticating against Kubernetes. Typically this is configured by spark-submit from" +
-        " mounting a secret from the submitting machine into the pod, and hence this" +
-        " configuration is marked as internal, but this can also be set manually to" +
-        " use a certificate that is mounted into the driver pod via other means.")
-      .internal()
-      .stringConf
-      .createOptional
-
-  private[spark] val KUBERNETES_DRIVER_MOUNTED_OAUTH_TOKEN =
-    ConfigBuilder(s"$APISERVER_DRIVER_CONF_PREFIX.mounted.oauthTokenFile")
-      .doc("Path on the driver pod's disk containing the OAuth token file to use when" +
-        " authenticating against Kubernetes. Typically this is configured by spark-submit from" +
-        " mounting a secret from the submitting machine into the pod, and hence this" +
-        " configuration is marked as internal, but this can also be set manually to" +
-        " use a token that is mounted into the driver pod via other means.")
-      .internal()
-      .stringConf
-      .createOptional
+  private[spark] val APISERVER_AUTH_SUBMISSION_CONF_PREFIX =
+      "spark.kubernetes.authenticate.submission"
+  private[spark] val APISERVER_AUTH_DRIVER_CONF_PREFIX =
```

Comment:
Yes, they are now embedded into the code itself. See `DriverPodKubernetesCredentialsMounterImpl` for example.